src/client/stylesheets/modules/home/home.module.scss

```
@@ -7,22 +7,23 @@
 
 .leftContainer {
   display:flex;
-  padding-left: 50px;
+  max-width: 400px;
   flex-direction:column;
   align-items: center;
 }
 
 .rightContainer{
   color:#999999;
-  padding-left:150px;
+  max-width: 300px;
+  padding-left: 50px;
   font-size:19px;
 }
 
 .lowerContainer {
   padding-top:30px;
   display: flex;
   flex-direction: row;
-  justify-content: center;
+  justify-content: space-around;
```

Thanks for submitting this - everything looks good on this PR except for this line - notice the spacing between  "Developed by Webpack..." text and "Create Project" button is too close together with this change. We'll eventually need some additional refactors to make sure everything is better-suited for all views. At the moment, it's just WIP set up for browser view - as long as any proposed styling changes are still consistent with the mocks on browser view, we're solid. Lets switch this back to `justify-content: center` and we're good to go 🤘
- 
```suggestion
  justify-content: center;
```

---

Thanks, I will update that. Can I give a padding to right-container so that it is at some space with left-container to compansate for this?

---

Thanks for submitting again - the spacing with these changes is still not consistent with the mocks. Here's link to the mocks - [https://github.com/webpack/webpack-cli/issues/825#issuecomment-489470582](https://github.com/webpack/webpack-cli/issues/825#issuecomment-489470582) As long as we're staying consistent with the spacing in the mocks, changes to accommodate responsive layouts are welcomed. thanks, appreciate it